virttest/utils_test/libvirt.py

```diff
@@ -3665,6 +3665,33 @@ def check_qemu_cmd_line(content, err_ignore=False):
     return True
 
 
+def run_remote_cmd(cmd, params, remote_runner=None):
+    """
+    A function to run a command on remote host.
+
+    :param cmd: the command to be executed
+    :param params: the parameter for executing
+    :param remote_runner: a remote runner object on remote host
+
+    :return: CmdResult object
+    :return: remote.RemoteRunner object
+    """
+    if not remote_runner:
+        remote_ip = params.get("server_ip", params.get("remote_ip"))
+        remote_pwd = params.get("server_pwd", params.get("remote_pwd"))
+        remote_user = params.get("server_user", params.get("remote_user"))
+
+        remote_runner = remote.RemoteRunner(host=remote_ip,
+                                            username=remote_user,
+                                            password=remote_pwd)
+    cmdResult = remote_runner.run(cmd, ignore_status=True)
+    logging.debug("Remote runner run result:\n%s", cmdResult)
+    if cmdResult.exit_status:
+        raise exceptions.TestFail("Failed to run '%s' on remote: %s"
+                                  % (cmd, results_stderr_52lts(cmdResult).strip()))
```

Comment:
How about add ignore_status? Maybe some usage do not want testfail if status error.

Reply:
updated.